package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "flights-searcher-cli",
+  "name": "flights-search-cli",
```

Comment:
There shouldn't be a `package-lock.json` change without a corresponding change to `package.json`.